feature_tests/features/producer/producer-createDocumentPointer-failure.feature

```diff
@@ -661,3 +703,55 @@ Feature: Producer Create Failure Scenarios
       | issue_code        | VALIDATION_ERROR                                                                                                                                         |
       | issue_description | A parameter or value has resulted in a validation error                                                                                                  |
       | message           | ValidationError raised from Data Contract 'Validate Content Url:1' at 'content[0].attachment.url': 'not-a-url' does not match '^https*://(www.)*\\w+.*$' |
+
+  @integration-only
+  Scenario: Validate a Document Pointer of type Mental health crisis plan using the asid data contract with ssp and no asid
```

Comment:
for all of the soft-linked files we should do a `Scenario Outline` (as opposed to `Scenario`) so that each feature test is run on each pointer-type. Otherwise if we were to remove/change one of the files we'd have no coverage on the behaviour for each pointer type